src/status.c

```diff
@@ -199,6 +199,71 @@ status_restore(struct view *view)
 	clear_position(&view->prev_pos);
 }
 
+static bool
+status_branch_tracking_info(char *buf, size_t buf_len, const char *head,
+			    const char *remote)
+{
+	if (!string_nformat(buf, buf_len, NULL, "%s...%s",
+			    head, remote)) {
+		return false;
+	}
+
+	const char *tracking_info_argv[] = {
+		"git", "rev-list", "--left-right", buf, NULL
+	};
```

Comment:
Sorry I'm a little late to the party, but I happened to notice this while pulling master. You can actually do `git rev-list --left-right --count` to get just the two counts. Probably not a big deal, but it would save a little pipe and parsing overhead.